scout/adapter/mongo/institute.py

```diff
@@ -146,6 +148,12 @@ def update_institute(
         if cohorts is not None:
             updates["$set"]["cohorts"] = cohorts
 
+        if loqusdb_id is not None:
+            LOG.info(
+                "Updating loqusdb id for institute: {0} to {1}".format(internal_id, loqusdb_id)
+            )
```

Comment:
I prefer the modern string formatting. Is there any reason to go old school?

Reply:
Yes this is not old school string formatting but rather lazy formatting which is specific to the `logging` module in python. It is always advised to use when logging since it does not affect performance when the log message is not being printed, that is the expression will not be evaluated in that case. Also it is nice to be consistent and we use lazy formated log messages on all (most:) other places.